app/server/server.ts

```diff
@@ -1,7 +1,12 @@
+const config = require('@hmcts/properties-volume').addTo(require('config'));
+const setupSecrets = require('./services/setupSecrets');
```

Comment:
can't we import this or at least import just the single function that is there?

```suggestion
import { setupKeyVaultSecrets } from './services/setupSecrets';
```

alternatively we could change the function to be the default export and then do:
```suggestion
import setupKeyVaultSecrets from './services/setupSecrets';
```